engine/dlib/src/dlib/dirent.h

```diff
@@ -0,0 +1,1027 @@
+/*
```

Comment:
Added for enabling same functionality on windows, as other platforms.
This allowed enabling some tests running on windows.